### ISSUE TITLE
The Monero Ecosystem project is closing

### ISSUE THREAD

**Issue by erciccione** [monero-ecosystem/meta#78]
As it's clear from all the unresolved issues, i don't have much time to manage this project at the moment. The problem is that this was supposed to be a collective of developers/people helping each other and helping keep the project going.

It was disappointing to see that, even if new members kept joining and triage powers were extended to all members, i ended up being the only one taking care of the project with the help of @lafudoci.

Now, i think the project has become very useful. Community members often point at it when somebody want to help with development and it's recognized as a solid source of info about Monero.

To go back to the original point: between Haveno, my CCS and other stuff (offline and online) i'm being very busy recently and that affected my activity in this project, which as a result completely halted it's activity of adding new members to the community. The question is: 

Are people interested in participating to the Monero Ecosystem and helping keep things going? I'll try to dedicate more time to the Ecosystem in the next days, but i can't promise anything. If we want the project to stay alive, people need to start to actively contribute.

@monero-ecosystem/maintainers please let me know your thoughts.

**Comment by woodser** [comment]
I think it's useful to have this collection of projects.

Developers contribute to their own project often.

There are open issues for monero-ecosystem/meta, but that doesn't mean they're high priority or necessary for the success of the ecosystem or individual projects.

**Comment by erciccione** [comment]
@woodser the issue is strictly related to the management of the Monero Ecosystem project itself and doesn't affect the ongoing development of the single projects. If we want to add more projects, i need others beside me to be actively involved in the voting/transfer process, otherwise we stagnate. I agree adding more projects is not a priority, but making the project grow as a whole (in terms of number of projects and active developers) should not be set aside, in my opinion.

**Comment by emesik** [comment]
I've been still within the crypto industry but quite detached from Monero recently. That's not going to change within coming months. However, I think that Monero Ecosystem is a great idea and should be continued.

I'll do my best to devote some time and take part in upcoming votes. Can't promise more right now. For Python projects I could also do quick code reviews.

**Comment by lh1008** [comment]
I've also been quite busy and my time has reduced significantly. I'm still on for voting and for anything else related. Not sure what else to do besides helping decide whether other projects should join or not. Erciccione do you have a list of other things you have been taking care of for the monero-ecosystem organization?

**Comment by DangerOnTheRanger** [comment]
I'm not currently a maintainer on any of the ecosystem projects, but I'd like to become the maintainer/a co-maintainer of [the golang RPC client](https://github.com/monero-ecosystem/go-monero-rpc-client), if possible. There's a couple PRs that I think could be merged immediately (monero-ecosystem/go-monero-rpc-client#8, monero-ecosystem/go-monero-rpc-client#7) that would have a good impact on the project, but have been sitting there for an unfortunate amount of time. I'll also personally take steps to write a full test suite for the client, since it doesn't currently have one from the look of things. (This isn't contingent on me becoming a maintainer, of course - just me announcing that I'll do so.) As far as prior contributions to Monero go, right now monero-project/monero#7823 is all I have, but as my Github PR history should hopefully indicate, I'm reasonably well-versed in golang.

The Python RPC client looks much more actively developed, but I can also pitch in there with code reviews and general upkeep from time to time.

**Comment by erciccione** [comment]
Thanks everybody for your feedback. Closing this issue.

> I'm not currently a maintainer on any of the ecosystem projects, but I'd like to become the maintainer/a co-maintainer of the golang RPC client, if possible

The only person that is able to do that is @omani, since the repo it's theirs. Try to contact them. If that repo is abandoned we could consider removing it from the project and instead add your updated repo (if people agree to do so).

**Comment by omani** [comment]
hi @DangerOnTheRanger I just merged your PRs. I wasnt available for a long time and just noticed this discussion.

the go rpc client is not abandoned and is being actively used. fyi.

cheers.

**Comment by erciccione** [comment]
> the go rpc client is not abandoned and is being actively used. fyi.

Great to know. Thanks for the update :)

**Comment by detherminal** [comment]
Hey @erciccione I want to be a maintainer for this organization. How can I apply for it? Do you have discord?

**Comment by erciccione** [comment]
Thanks for offering @detherminal, but who gets maintainer powers has also full powers in the repositories hosted by the project. These are permissions that cannot be given lightly.

I don't see any other possibility then leaving things as they are, in some sort of maintainance mode, where repo maintainers can keep working on their projects, but new ones will probably not join the ecosystem.

**Comment by erciccione** [comment]
Reopening to let @monero-ecosystem/maintainers to invite them to move their repositories to their personal profiles or under other organizations. I'm moving forward with closing this organization. Unless somebody reliable wants the responsibility of these repositories.

**Comment by Lafudoci** [comment]
Thank you @erciccione for these years, the responsibility to keep thing on rail is not easy. It seems the lack of enough active members/reviewers has made governance system unable to run well. But I am still thinking about your previous comment, leaving things as they are, like a simple archived repo collection. Maybe we could remove the governance part to reduce responsibility loading of the organization maintainer? I believe it is still worth to keep this monero projects collection.

**Comment by erciccione** [comment]
Pinging all maintainers: @Lafudoci @rehrar @starmute @ajs-xmr @dougbebber @emesik @NoThisIsPatrik @xmrdsc @normoes @ehanoc @woodser @Jasonhcwong @omani @shermand100 @Agorist-Action

**Comment by erciccione** [comment]
Reminder: The Ecosystem project will be archived soon. Please move your repositories to your personal accounts or to another Github project.

**Comment by normoes** [comment]
Thanks for everything.

**Comment by erciccione** [comment]
Last ping before all remaining repositories will get archived.

**Comment by erciccione** [comment]
I'm closing this issue, the project and archiving all remaining repositories. Thank you for participating to this experiment, too bad Monero has changed and became unrecognisable from the small principle-driven community it once was. Good luck to everybody.
